test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua

```diff
@@ -0,0 +1,170 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [PTU] [GENIVI] PolicyTableUpdate has NO "certificate" and "ForceProtectedService"=OFF at .ini file
+-- [PTU] [GENIVI] SDL must start PTU for any app except navi right after app successfully request to start first secure service
```

Comment:
@dboltovskyi, thank you for notice. Done in 5ba86daa08d2a7bdfd485dacc011a2d550964b1a